packages/loot-core/src/server/accounts/transaction-rules.ts

```
@@ -849,6 +876,14 @@ export async function prepareTransactionForRules(
     }
   }
 
+  if (trans.account) {
+    if (accounts !== null && accounts.has(trans.account)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)